vllm/model_executor/models/llama.py

```diff
@@ -93,12 +94,20 @@ def __init__(
         assert self.total_num_heads % tensor_model_parallel_world_size == 0
         self.num_heads = (self.total_num_heads //
                           tensor_model_parallel_world_size)
+        self.total_num_key_value_heads = num_key_value_heads
+        assert (self.total_num_key_value_heads %
+                tensor_model_parallel_world_size == 0)
+        self.num_key_value_heads = (self.total_num_key_value_heads //
+                                    tensor_model_parallel_world_size)
```

Comment:
nit: I feel the variable names are unnecessarily long and they are making the code less pretty. Can we use something like `num_kv_heads` and `tp_size` instead?